near-memory-tracker/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod allocator;
```

Comment:
Just like with the other profiler, let's put it into the `utils/near-memory-tracker`

Reply:
I actually suggest that we put this in a separate repo. There is nothing nearcore specific here

Reply:
I'll put it in a separate repo. There are still some features I want to add to debug memory leaks.